toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -1,5 +1,5 @@
 filesystem-1.1-15.cm2.aarch64.rpm
-kernel-headers-5.15.131.1-1.cm2.noarch.rpm
+kernel-headers-5.15.131.1-2.cm2.noarch.rpm
```

Comment:
Touching the kernel spec might require a bit of modification for a bunch of files.
e.g `kernel-headers` is a spec that needs to be in sync with the version-release of kernel.

A representative PR: 
- https://github.com/microsoft/CBL-Mariner/pull/6196